src/tigerbeetle/cli.zig

```diff
@@ -258,6 +260,9 @@ pub fn parse_args(allocator: std.mem.Allocator) !Command {
                     .{ storage_size_limit, constants.sector_size },
                 );
             }
+            if (start.in_memory and !std.mem.eql(u8, start.positional.path, "in-memory")) {
```

Comment:
Actually, let me take my :+1: back here -- I think we would want our users to use this option in their tests, no? So this would be our official surface area.

Perhaps this wants to be a separate command even, as this is actually `format-and-start`?